website/src/pages/index.tsx

```
@@ -19,14 +19,30 @@ function Hero(): JSX.Element {
-              Best Free &amp; Open Source Tool for Containers &amp; Kubernetes
-            </h1>
-            <p className="text-base font-bold leading-[1.9rem] md:text-lg md:leading-[1.9rem] mt-[88px]">
-              The best free and open source tool for developers to work with containers and Kubernetes. Simplify
-              container management, streamline Kubernetes workflows, and transition from local development to production
-              with ease.
-            </p>
+            <div className="relative">
+              <img
+                className="absolute -scale-x-100 right-0 -top-23 w-22 animate-float"
+                src="img/3MBanner/party-seal.png"
+              />
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)